src/column-layout/styles.scss

```diff
@@ -130,5 +131,49 @@ div.column-layout {
     &.grid-horizontal-borders {
       @include make-horizontal-borders;
     }
+
+    // Alternative implementation based on minimum column widths and CSS Grid
+    &.css-grid {
```

Comment:
How about extracting new styles into separate file, to have it properly co-located CSS and JS?

Reply:
I could do that, but I'm also reusing a good amount of styles/classes from the main implementation. Mainly things around borders. So I would still need to import those classes.

Reply:
Your call, but if there are not too many classes, I would copy them instead. The "overlay thinking" where I need to add up styles from two places together makes it harder to read and prone to bugs